index.html

```diff
@@ -26,28 +26,29 @@ <h1>
          <li><a href="#"><img src="cart.jpg" alt="shopping cart logo">shopping cart (2)</a></li>
       </ul>
       <ul>
-        <img src="shoe2.png" alt="picture of blue and orange sneaker">
-        <img src="shoe3.png" alt="picture of red and black sneaker">
-        <img src="shoe4.png" alt="picture of blue and white sneaker">
+        <img src="shoe2.png" alt="picture of blue and orange sneaker" class="shoe2">
+        <img src="shoe3.png" alt="picture of red and black sneaker" class="shoe3">
+        <img src="shoe4.png" alt="picture of blue and white sneaker" class="shoe4">
     </ul>
     </header>
 
 
-    <img src="shoe1.png" alt="picture of black and white sneaker">
-
+    <img src="shoe1.png" alt="picture of black and white sneaker" class="shoe1">
 
+    <footer>
+      <ul class="left">
+        <li><img class="shoelogo" src="logo.jpg" alt="shoelogo"></li>
+        <li><a href="#">MEN</a></li>
+        <li><a href="#">WOMEN</a></li>
+        <li><a href="#">BOYS</a></li>
+        <li><a href="#">GIRLS</a></li>
+        <li><a href="#">CUSTOMIZE</a></li>
+      </ul>
+      <ul class="right">
+        <li><a href="#">Shipping|</a></li>
+        <li><a href="#">terms & conditions|</a></li>
+        <li><a href="#">@ Copyright 2017.Sneaker Shop,LLC.</a></li>
+      </ul>
+    </footer>
```

Comment:
I moved the `<footer>` element up into the `<body>` element as well. Anything you want people to see on your webpage should be inside the `<body>`. Sometimes you can still see content that you've accidentally put outside the `<body>`, but it may not act like you expect it to!